src/Mailer/Transport/DebugKitTransport.php

```diff
@@ -36,7 +37,7 @@ class DebugKitTransport extends AbstractTransport
      */
     public function __construct($config = [], ?AbstractTransport $originalTransport = null)
     {
-        $this->emailLog = $config['debugKitLog'];
+        $this->emailLog = $config['debugKitLog'] ?? new ArrayObject();
```

Comment:
Will this still capture email messages? I think the email panel relies on passing the log object in.